### PR TITLE
[GNA] [MTL] [North] [2022.1.0] Engineering build for PO

### PR DESCRIPTION
**Pull request by kbruniec**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/inference/include/ie/gna/gna_config.hpp [hunk 1]
```diff
@@ -183,6 +183,8 @@ DECLARE_GNA_CONFIG_KEY(EXEC_TARGET);
 DECLARE_GNA_CONFIG_VALUE(TARGET_2_0);
 DECLARE_GNA_CONFIG_VALUE(TARGET_3_0);
 
+DECLARE_GNA_CONFIG_VALUE(TARGET_3_5);
```
Comment (nkogteva) [inline src/inference/include/ie/gna/gna_config.hpp:186]: Please don't touch old config, for new options use new one only

File: CMakeLists.txt [hunk 2]
```diff
@@ -2,6 +2,11 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if(WIN32)
+    message(STATUS "Appending Zi flag")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Zi")
+endif()
```
Comment (ilya-lavrenov) [inline CMakeLists.txt:9]: why do we need this?

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**